### PR TITLE
Grammar fix / improvements

### PR DESCRIPTION
**Pull request by Blank3495**
as usual, check them :P

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
 [![Donate](https://img.shields.io/badge/Donate-PayPal-blue.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=TSTEG3PXK4HJ4&source=url)
 
-Funny or not, donations are fundamental to keep the development of the project.
+Funny or not, donations are fundamental to keep the development of the project going.
 
-If you have disponibility to donate any value, do not hesitate in helping us on PayPal!
+If you are able to donate any value, do not hesitate in helping us via PayPal!
 
-###### Thanks for all the **$22** of donations until now! ♥
+###### Thanks for all the donations until now! **($22)** ♥
```
[the remark this answers is not in the document]
Comment (Blank3495) [reply CONTRIBUTING.md]: > I don't like this one.

i like the quote
if u put $22 in the middle we need to rephrase the whole sentence

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -3,11 +3,11 @@
 ## Donations
 [![Donate](https://img.shields.io/badge/Donate-PayPal-blue.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=TSTEG3PXK4HJ4&source=url)
 
-Funny or not, donations are fundamental to keep the development of the project.
+Funny or not, donations are fundamental to keep the development of the project going.
 
-If you have disponibility to donate any value, do not hesitate in helping us on PayPal!
+If you are able to donate any value, do not hesitate in helping us via PayPal!
```
[the remark this answers is not in the document]
Comment (Blank3495) [reply CONTRIBUTING.md]: not many people know the meaning of it, and `are able` makes more sense/understandable/looks cute *-*

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Lautenschlager-id) [reply CONTRIBUTING.md]: Okay, ignore it then.

File: CONTRIBUTING.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Lautenschlager-id) [reply CONTRIBUTING.md]: I prefer disponibility

[the remark this answers is not in the document]
Comment (Blank3495) [reply CONTRIBUTING.md]: alright rewrite the sentence then

**Pull request merged**